### DISCUSSION TITLE
Which 🎥 starred Tom Cruise and Dustin Hoffman?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1054] [Galaxy Trivia] [answered]
**Bonus:** Which real-life scientist is the movie based on?

**Comment by peterbaccaro** [comment]
The film that starred Tom Cruise and Dustin Hoffman together is called **Rain Man**. The movie was released in 1988 and was directed by Barry Levinson. In the film, Tom Cruise plays Charlie Babbitt, a self-centred car dealer, and Dustin Hoffman plays Raymond Babbitt, Charlie's autistic savant brother. The movie received critical acclaim and won several awards, including four Academy Awards, including Best Picture, Best Director, Best Original Screenplay, and Best Actor for Dustin Hoffman's exceptional performance as Raymond.

The character of Raymond Babbitt in the movie Rain Man was inspired by a real-life person named **Kim Peek**.
Kim Peek was an extraordinary individual known as a "mega-savant." He was born in 1951 and had a remarkable memory and an incredible ability to absorb and recall vast amounts of information. He had developmental disabilities, and his condition was later diagnosed as savant syndrome. Kim Peek's abilities were the inspiration behind the character of Raymond Babbitt, portrayed by Dustin Hoffman in the film "Rain Man." While the movie is fictional and not a biographical account of Kim Peek's life, it was influenced by Peek's unique abilities and the real-life experiences of Kim and his father, Fran Peek.

**Comment by thuongtruong1009** [comment, marked as answer]
The movie that starred Tom Cruise and Dustin Hoffman is "Rain Man," released in 1988. In the film, Tom Cruise plays the character Charlie Babbitt, a self-centered car dealer, while Dustin Hoffman portrays his autistic savant brother, Raymond Babbitt.

The movie "Rain Man" was not directly based on a real-life scientist. Instead, it explores the relationship between the two brothers as they embark on a cross-country journey. However, the character of Raymond Babbitt was inspired by the experiences of real-life savant Kim Peek, who had remarkable memory and cognitive abilities despite being diagnosed with severe developmental disabilities. Peek's abilities were attributed to a condition called savant syndrome.

Though the character was not directly based on a specific scientist, the film brought awareness to the unique abilities and challenges faced by individuals with savant syndrome. Dustin Hoffman's portrayal of Raymond Babbitt earned him critical acclaim and won him the Academy Award for Best Actor.
